index.ts

```diff
@@ -67,7 +67,13 @@ client.on("message", (message: Message) => {
   }
   if (
     message.content.toLowerCase().includes("matt") ||
-    message.author.id === "541305895544422430"
+    message.author.id === "541305895544422430" ||
+    message.content.includes(
+      ":regional_indicator_m::regional_indicator_a::regional_indicator_t::regional_indicator_t:"
+    ) ||
+    message.content.includes(
+      ":regional_indicator_m::a::regional_indicator_t::regional_indicator_t:"
+    )
```

Comment:
```suggestion
    message.content.includes("🇲 🇦 🇹 🇹") ||
    message.content.includes("🇲 🅰️ 🇹 🇹")
```
Were you able to get the `:regional_indicator:` strings to work locally? I pulled this down and it wouldn't work - Discord does some funko stuff behind the scenes with emoji. 
However, I *did* get it to recognise the message if I used the `unicode` emoji here in the code instead of the Discord call string.

Reply:
Oh, I didn't think of that bro. Well, IDK if UNICODE works, but yeah. I'll look on to it...

And don't close this PR, I'll do the changes later tomorrow in my Time Zone as I'm so tired lol.

Reply:
The unicode here in `.includes` provided the functionality on my local instance. 😁 

No worries, get some sleep!